_data/reference.yml

```diff
@@ -49,20 +49,20 @@ toc:
 
 - title: kubectl CLI
   section:
-  - docs/user-guide/kubectl-overview.md
-  - docs/user-guide/kubectl/index.md
+  - docs/reference/generated/kubectl-overview.md
```

Comment:
I think this is not generated.

Reply:
@tengqm Yeah, I will split it into serveral commits. https://github.com/kubernetes/website/issues/6037 said that the /docs/user-guide/ directory will be  deprecated but I do not know where put these file. May be we should create a directory somewhere.